### DISCUSSION TITLE
How to paint terrain?

### DISCUSSION THREAD

**Discussion by nanodeath** [TokisanGames/Terrain3D#292] [Questions] [answered]
This seems like something that should be super basic, but I can't seem to get it to work. See the video:

https://github.com/TokisanGames/Terrain3D/assets/10625/efd6aa18-9222-454d-8065-fa1033349707

**Comment by TokisanGames** [comment, marked as answer]
Disable material/debug views/colormap. You imported a colormap like a satellite image which enables that view in the material for GIS applications. When disabled the colormap will be multiplied with albedo. 

Do not use the Texture list for a satellite image either. Use it for paintable textures of a singular material like grass. 

Watch the tutorial video linked on the front page and read through the documentation.

**Reply by nanodeath** [reply]
Okay, but I don't want to do that -- I want to use the existing texture as a guide.

**Reply by TokisanGames** [reply]
That's how it works now. Edit the shader if you want it to work different.

If you turn off the debug view you can see the colormap as a guide, multiplied into albedo.

Either way it paints fine and is working as designed.
